fs/dafilesrv/dafilesrv.cpp

```diff
@@ -396,16 +395,8 @@ int main(int argc, const char* argv[])
 #endif
 
     const char* componentTag = "dafilesrv";
-    Owned<IPropertyTree> componentDefault;
-    if (defaultYaml)
-    {
-        Owned<IPropertyTree> defaultConfig = createPTreeFromYAMLString(defaultYaml, 0, ptr_ignoreWhiteSpace, nullptr);
-        componentDefault.set(defaultConfig->queryPropTree(componentTag));
-        if (!componentDefault)
-            throw makeStringExceptionV(99, "Default configuration does not contain the tag %s", componentTag);
-    }
-    else
-        componentDefault.setown(createPTree(componentTag));
+    Owned<IPropertyTree> defaultConfig = createPTreeFromYAMLString(defaultYaml, 0, ptr_ignoreWhiteSpace, nullptr);
+    Owned<IPropertyTree> componentDefault(defaultConfig->queryPropTree(componentTag));
```

Comment:
-ve leak. Needs to be a getPropTree

Reply:
Ok, right, I need to pay more attention to query/get and set/setown, it is slightly unituitive to me for some reason. So, is the following correct: get* methods will increase the reference count, query* methods will not. Owned constructor DOES not increase reference count, set increases reference count while setown does not.

Reply:
correct, and Linked would also increase the link count (and both Owned and Linked will descrease it).
So:
```
Linked<IPropertyTree> componentDefault(defaultConfig->queryPropTree(componentTag));
```
would also fix the -ve leak, i.e. Linked increasing the link count on ctor, and decreasing it on dtor.